05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,39 @@
-.lawn {
-    width: 100%;
-    height: 300px;
-}
+/*CSS for Berries*/
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+.image {
+  width:500px;
+  float:left;
 }
 
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.image img {
+  width:100%;
+  height:100%;
 }
 
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+.text {
+  width:500px;
+  height:398px;
+  float:right;
```

Comment:
float this guy left as well. we haven't covered floating yet, but once we do it's gonna make sense.